macros/upload_sources.sql

```diff
@@ -57,11 +63,14 @@
                     '{{ source.database }}', {# database #}
                     '{{ source.schema }}', {# schema #}
                     '{{ source.source_name }}', {# source_name #}
+                    '{{ source.source_description }}', {# source_description #}
                     '{{ source.loader }}', {# loader #}
                     '{{ source.name }}', {# name #}
                     '{{ source.identifier }}', {# identifier #}
                     '{{ source.loaded_at_field | replace("'","\\'") }}', {# loaded_at_field #}
-                    parse_json('{{ tojson(source.freshness) | replace("'","\\'") }}')  {# freshness #}
+                    parse_json('{{ tojson(source.freshness) | replace("'","\\'") }}'),  {# freshness #}
+                    parse_json('{{ tojson(source.source_meta) | replace("'","\\'") }}'),  {# source_meta #}
+                    parse_json('{{ tojson(source.meta) | replace("'","\\'") }}')  {# meta #}
```

Comment:
Please can you add examples of these configs to the sources.yml for the integration tests? https://github.com/brooklyn-data/dbt_artifacts/blob/main/integration_test_project/models/sources.yml